demo/laneLoader.js

```diff
@@ -528,21 +551,78 @@ function updateSegments(laneSegments, clonedBoxes, prevIsContains, point, index,
   return newIsContains
 }
 
+
+// Adds anomaly annotations
+function addAnnotations(laneGeometries) {
+  let point;
+
+  let aRoot = viewer.scene.annotations;
+  let aAnomalies = new Potree.Annotation({
+    title: "Lane Anomalies",
+    position: laneGeometries.leftAnomalies[0]
+  });
+  aAnomalies.visible = false;
+  aRoot.add(aAnomalies);
+  let aLeft = new Potree.Annotation({
+    title: "Left",
+    position: laneGeometries.leftAnomalies[0]
+  });
+  let aRight = new Potree.Annotation({
+    title: "Right",
+    position: laneGeometries.rightAnomalies[0]
+
+  });
+  aAnomalies.add(aLeft);
+  aAnomalies.add(aRight);
+
+  // Add anomalies layer
+  for (let ii=0, len=laneGeometries.leftAnomalies.length; ii<len; ii++) {
+    point = laneGeometries.leftAnomalies[ii];
+    let aAnomaly = new Potree.Annotation({
+      title: "L " +[ii+1],
+      position: point,
+      cameraPosition: new THREE.Vector3(point.x, point.y, point.z + 20),
+      cameraTarget: point
+    });
+    aLeft.add(aAnomaly);
+  }
+  for (let ii=0, len=laneGeometries.rightAnomalies.length; ii<len; ii++) {
+    point = laneGeometries.rightAnomalies[ii];
+    let aAnomaly = new Potree.Annotation({
+      title: "R " +[ii+1],
+      position: point,
+      cameraPosition: new THREE.Vector3(point.x, point.y, point.z + 20),
+      cameraTarget: point
+    });
+    aRight.add(aAnomaly);
+  }
+}
+
+
+// Adds lane geometries to viewer
+function addLaneGeometries(laneGeometries, lanesLayer) {
+
+  for (let ii = 0, len = laneGeometries.all.length; ii < len; ii++) {
+    lanesLayer.add(laneGeometries.all[ii]);
+  }
+  viewer.scene.scene.add(lanesLayer);
+  
+  // add lane anomaly geometries
+  if (laneGeometries.leftAnomalies.length != 0 || laneGeometries.rightAnomalies.length != 0) {addAnnotations(laneGeometries);}
+}
+
+
 // Load Lanes Truth Data:
 export async function loadLanesCallback(s3, bucket, name, callback) {
 
 	let filename, tmpSupplierNum;
 	tmpSupplierNum = -1;
 	await loadLanes(s3, bucket, name, filename, tmpSupplierNum, window.annotateLanesModeActive, viewer.scene.volumes, (laneGeometries) => {
-
 		// need to have Annoted Lanes layer, so that can have original and edited lanes layers
 		let lanesLayer = new THREE.Group();
 		lanesLayer.name = "Lanes";
-		for (let ii = 0, len = laneGeometries.all.length; ii < len; ii++) {
-			lanesLayer.add(laneGeometries.all[ii]);
-		}
-		viewer.scene.scene.add(lanesLayer);
-		viewer.scene.dispatchEvent({
+    addLaneGeometries(laneGeometries, lanesLayer);
```

Comment:
Can you fix the changes in indenting also please; occurs in more than one place

Reply:
not sure what's happening, but it doesn't look like this on my local machine. will try to fix

Reply:
Usually when there are discrepancies like this in whitespace, it's spaces vs. tabs. Don't use tabs. There is probably a setting in your editor or IDE for this.